core/lib/zksync_core/src/consistency_checker/mod.rs

```diff
@@ -248,7 +252,11 @@ impl ConsistencyChecker {
             .await?)
     }
 
-    pub async fn run(mut self, mut stop_receiver: watch::Receiver<bool>) -> anyhow::Result<()> {
+    pub async fn run(
+        mut self,
+        mut stop_receiver: watch::Receiver<bool>,
+        l1_batch_commit_data_generator: Arc<dyn L1BatchCommitDataGenerator>,
```

Comment:
Please add parameters to the `ConsistencyChecker` structure itself, rather than to the `run()` method.

Reply:
Tracked in https://github.com/lambdaclass/zksync-era/issues/132